DataHubClient.js

```diff
@@ -234,14 +242,14 @@ export class DataHubClient {
     // decrypt doc content
     const {cipher, kek} = this;
     const {jwe} = encryptedDoc;
-    const content = await cipher.decryptObject({jwe, kek});
-    if(content === null) {
+    const {content, meta} = await cipher.decryptObject({jwe, kek});
+    if(content === null || meta === null) {
       throw new Error('Decryption failed.');
```

Comment:
`decryptObject` should return `null` when it fails, not an object with `content: null, meta: null` -- then check for that ... then destructure as needed and return.